commandapi-platforms/commandapi-paper/commandapi-paper-nms/commandapi-paper-nms-common/src/main/java/dev/jorel/commandapi/nms/PaperNMS_CommonWithFunctions.java

```diff
@@ -0,0 +1,5 @@
+package dev.jorel.commandapi.nms;
+
+public abstract class PaperNMS_CommonWithFunctions extends PaperNMS_Common {
+
+}
```

Comment:
This class doesn't seem necessary. Instead of extending `PaperNMS_CommonWithFunction`, subclasses could just extend `PaperNMS_Common` directly. `dev/paper/restructure-experiment` has [done this](https://github.com/JorelAli/CommandAPI/commit/666a422416948bc756ed6331768563da74a83e02#diff-7672b0644b015eb0e0888ed2568021f5e3c2fd9b369cff243753ed72707a36d4L1-L5) without any problems.